src/mongo/util/net/message_port.cpp

```diff
@@ -184,13 +186,6 @@ namespace mongo {
                 send( s.c_str(), s.size(), "http" );
                 return false;
             }
-            else if ( len == -1 ) {
-                // Endian check from the client, after connecting, to see what mode server is running in.
-                unsigned foo = 0x10203040;
```

Comment:
Most likely no, but it is generally easier to keep the code as close as possible, so that someday down the road we don't end up with some sort of merge conflict here and then need to do a bunch of archaeology.